monitor.js

```diff
@@ -62,8 +62,10 @@ function checkStatus() {
     if (!started) {
       started = true;
       // Trigger tests
-      axios.post(fis[0].url);
-      axios.post(fis[1].url);
+      axios.post(fis[0].url, { assetCode: 'XLM' });
+      axios.post(fis[1].url, { assetCode: 'XLM' });
+      axios.post(fis[0].url, {assetCode: 'TEST'});
+      axios.post(fis[1].url, { assetCode: 'TEST' });
```

Comment:
See my other comment. Call to `/tests` should start all tests in a single FI, so no changes needed in this file.